.github/workflows/build-deploy-api-v2.yaml

```diff
@@ -167,15 +167,31 @@ jobs:
     environment: API-Dev
     needs: [ DEV-cleanup ]
     steps:
+      - name: Message Teams Channel on Success
+        run: |
+          curl --location --request POST ${{ secrets.TEAMS_SUCCESS_URL }} \
+          --header 'Content-Type: text/plain' \
+          --data-raw '{
+            "@context": "https://schema.org/extensions",
+            "@type": "MessageCard",
+            "themeColor": "0072C6",
+            "title": "SUCCESS! Deployment to PIA ${{ secrets.ENVIRONMENT }} Environment has finished.",
+            "text": "The PIA API has been built and deployed to the ${{ secrets.ENVIRONMENT }} environment. \n\n The logs for the successful deployment can be found [here](${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }})\n\n The changes for commit ${{ github.sha }} can be found [here](https://dev.pia.gov.bc.ca/api/api-docs).",
```

Comment:
For the body, We can add the git commit message description. A habit which should be encouraged which states what changes this commit has, in addition to what you have.

Reply:
See above.